client/package.json

```diff
@@ -52,20 +52,20 @@
     "zone.js": "0.15.1"
   },
   "devDependencies": {
-    "@angular-eslint/builder": "19.8.0",
-    "@angular-eslint/eslint-plugin": "19.8.0",
-    "@angular-eslint/eslint-plugin-template": "19.8.0",
-    "@angular-eslint/schematics": "19.8.0",
-    "@angular-eslint/template-parser": "19.8.0",
-    "@angular/build": "20.0.1",
-    "@angular/cli": "20.0.1",
-    "@angular/compiler-cli": "20.0.1",
-    "@eslint/js": "9.28.0",
-    "@types/jest": "29.5.14",
-    "@typescript-eslint/eslint-plugin": "8.33.1",
-    "@typescript-eslint/parser": "8.33.1",
-    "eslint": "9.28.0",
-    "jest": "29.7.0",
+    "@angular-eslint/builder": "20.1.1",
+    "@angular-eslint/eslint-plugin": "20.1.1",
+    "@angular-eslint/eslint-plugin-template": "20.1.1",
+    "@angular-eslint/schematics": "20.1.1",
+    "@angular-eslint/template-parser": "20.1.1",
+    "@angular/build": "20.1.1",
+    "@angular/cli": "20.1.1",
+    "@angular/compiler-cli": "20.1.1",
+    "@eslint/js": "9.31.0",
+    "@types/jest": "30.0.0",
+    "@typescript-eslint/eslint-plugin": "8.37.0",
+    "@typescript-eslint/parser": "8.37.0",
+    "eslint": "9.31.0",
+    "jest": "30.0.4",
```

Comment:
use highest Angular compatible version for Jest, okay if not latest

ensure build and test passes